src/lib/Makefile.am

```diff
@@ -20,7 +20,6 @@ librnp_la_SOURCES	= \
 	key_store.c \
 	key_store_pgp.c \
 	key_store_kbx.c \
-	key_store_ssh.c \
```

Comment:
@ni4 I see. Looks like I just missed it :)

Anyway, I agree that ssh keysotre looks like an orphan and it requests complete redisign because it has (had?) hardcoded 'id_rsa'.

Can you just make it build optional for now over confgure flag?